<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalSettingsDialog</class>
 <widget class="QDialog" name="GlobalSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentRow">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>General</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Conversation</string>
        </property>
        <property name="icon">
         <iconset resource="../../icons.qrc">
          <normaloff>:/data/gui/send.png</normaloff>:/data/gui/send.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Features</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Network</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="VerticalScrollArea" name="scrollArea">
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>354</width>
          <height>397</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="pageGeneral">
            <layout class="QFormLayout" name="formLayout">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="1" column="0">
              <widget class="QLabel" name="showSystrayLabel">
               <property name="text">
                <string>Show on systray:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="showSystrayCheckBox"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="languageLabel">
               <property name="text">
                <string>Language:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="languageComboBox"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageConversation">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QGroupBox" name="emoticonsGroupBox">
               <property name="title">
                <string>Emoticons</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QCheckBox" name="emoticonsEnabledCheckBox">
                  <property name="text">
                   <string>Enabled</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="emoticonsLayout">
                  <item>
                   <widget class="QWidget" name="emoticonsWidget" native="true">
                    <layout class="QHBoxLayout" name="horizontalLayout_3">
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QTableView" name="emoticonsView">
                       <property name="selectionMode">
                        <enum>QAbstractItemView::SingleSelection</enum>
                       </property>
                       <property name="selectionBehavior">
                        <enum>QAbstractItemView::SelectRows</enum>
                       </property>
                       <property name="showGrid">
                        <bool>false</bool>
                       </property>
                       <property name="gridStyle">
                        <enum>Qt::NoPen</enum>
                       </property>
                       <attribute name="verticalHeaderVisible">
                        <bool>false</bool>
                       </attribute>
                      </widget>
                     </item>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_6">
                       <item>
                        <spacer name="verticalSpacer">
                         <property name="orientation">
                          <enum>Qt::Vertical</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>20</width>
                           <height>40</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                       <item>
                        <widget class="QPushButton" name="upButton">
                         <property name="minimumSize">
                          <size>
                           <width>32</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>32</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="toolTip">
                          <string>Move up</string>
                         </property>
                         <property name="text">
                          <string>▲</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QPushButton" name="downButton">
                         <property name="minimumSize">
                          <size>
                           <width>32</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="maximumSize">
                          <size>
                           <width>32</width>
                           <height>32</height>
                          </size>
                         </property>
                         <property name="toolTip">
                          <string>Move down</string>
                         </property>
                         <property name="text">
                          <string>▼</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <spacer name="verticalSpacer_2">
                         <property name="orientation">
                          <enum>Qt::Vertical</enum>
                         </property>
                         <property name="sizeHint" stdset="0">
                          <size>
                           <width>20</width>
                           <height>40</height>
                          </size>
                         </property>
                        </spacer>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageFunctions">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QGroupBox" name="spellCheckBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="title">
                <string>Spell Checking</string>
               </property>
               <layout class="QFormLayout" name="formLayout_2">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="enableLabel">
                  <property name="text">
                   <string>Enable</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QCheckBox" name="enableCheckBox"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="spellCheckLanguageLabel">
                  <property name="text">
                   <string>Language:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="spellCheckLanguageComboBox"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="highlightGroupBox">
               <property name="title">
                <string>Highlighting</string>
               </property>
               <layout class="QFormLayout" name="formLayout_3">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="highlightSpellingMistakesLabel">
                  <property name="text">
                   <string>Highlight spelling mistakes:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QCheckBox" name="highlightSpellingMistakesCheckBox"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="highlightCommandsLabel">
                  <property name="text">
                   <string>Highlight commands:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QCheckBox" name="highlightCommandsCheckBox"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="pageNetwork"/>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VerticalScrollArea</class>
   <extends>QScrollArea</extends>
   <header>gui/verticalscrollarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GlobalSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GlobalSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>emoticonsEnabledCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>emoticonsWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
